PurchasesCoreSwift/Public/ReOrgLater/EntitlementInfoEnums.swift

```diff
@@ -1,39 +0,0 @@
-//
-//  EntitlementInfoEnums.swift
-//  PurchasesCoreSwift
-//
-//  Created by Joshua Liebowitz on 6/24/21.
```

Comment:
is the intent of moving this into the other file to keep the same file structure that we had before? I actually liked having these separate, since the other file is subject to grow

Reply:
The intent is to maintain the same file structure as before. There's enough moving pieces I want to be sure our first pass is just migration. We definitely should consider organization as an optimization step 😄